origin-faucet/faucet/app.js

```diff
@@ -88,10 +95,14 @@ function runApp(config) {
 const args = Config.parseArgv()
 const config = {
   // Port server listens on.
-  port: args['--port'] || DEFAULT_SERVER_PORT,
+  port: args['--port'] || process.env.PORT || DEFAULT_SERVER_PORT,
```

Comment:
Should we use ParseInt here ?
Same comment for networkIds

Not in this PR but we should really find a good npm package for handling argument parsing... :)